charts/superset/templates/sqlite-pvc.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "superset.fullname" . }}-sqlite-pvc
+  labels:
+    {{- include "superset.labels" . | nindent 4 }}
+spec:
+  persistentVolumeReclaimPolicy: Retain
```

Comment:
I can't apply the chart due to this:
```
  Error: Failed to render chart: exit status 1: Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(PersistentVolumeClaim.spec): unknown field "persistentVolumeReclaimPolicy" in io.k8s.api.core.v1.PersistentVolumeClaimSpec, ValidationError(PersistentVolumeClaim.spec): unknown field "storageClass" in io.k8s.api.core.v1.PersistentVolumeClaimSpec]
  Error: plugin "diff" exited with error
```

I wonder if you managed to? I think this key is only valid on the PV not the PVC but in any case you might rather add this to the storageclass (or instead use a stateful set in place of the deployment). See this S/O post for some thoughts https://stackoverflow.com/questions/50458165/kubernetes-how-to-do-dynamic-persistentvolumeclaim-with-persistentvolumereclaim